modules/github-runners/main.tf

```diff
@@ -52,11 +51,13 @@ data "cloudinit_config" "config" {
     content_type = "text/x-shellscript"
     filename     = "user-data.sh"
     content = templatefile(local.userdata_template, {
-      github_token_ssm_path = local.github_token_ssm_path
-      registration_url      = local.registration_url
-      runner_version        = var.runner_version
-      runner_name_prefix    = module.this.id
-      runner_labels         = var.runner_labels
+      docker_compose_version = var.docker_compose_version
+      github_token_ssm_path  = join("", data.aws_ssm_parameter.github_token.*.name)
+      github_scope           = var.github_scope
```

Comment:
The latest runner is in `infra-live` and uses multiple github scopes. Could you reconcile this upstream with the component updates in `infra-live` ?

cc: @goruha